prov/tcp/src/xnet.h

```diff
@@ -69,6 +69,7 @@
 
 #define XNET_RDM_VERSION	0
 #define XNET_DEF_INJECT		128
+#define XNET_DEF_MAX_RX		16384
```

Comment:
We'll need to figure out the correct size here.  If the receiver is slow posting buffers, a larger size allows buffering the message, but results in more data being copied twice.  I'd rename, see comment below.